content/blog/2023-01-25-ethereums-social-consensus-vs-ethereum-classics-code-is-law/index.md

```diff
@@ -0,0 +1,151 @@
+
+It was the enormous amount of electricity needed, the enormous amount of computational power required, the difficulty adjustment, and the importation of this information from the real physical world into the system that was the revolutionary solution to the problem of consensus and safety in the network.
+
+Social Consensus' lack of acknowledgement of this truth and their naive refocusing on human virtue as a security measure is the major flaw of that philosophy.
+
+This flaw makes them believe in several falsehoods, one of which is that a blockchain is a sort of "crypto-economic organism" or some sort of righteous organization of people who will always be good for the world.
+
+Social Consensus focuses on economics and virtue rather than physics and energy.
+
+Social Consensus glorifies "the community", considers it the security of last resort, and believes that human voluntary agreement is what ultimately protects the accounts, property, dapps, and contracts inside a blockchain.
+
+We would argue, that a Social Consensus network is not even a blockchain. Blocks are defined as batches of data that have to be sealed with a proof of work hash that must cost block producers enormous amounts of capital and energy.  
+
+The ultimate purpose of Social Consensus is to protect the planet, to do no harm to the environment, and to heal the climate. The whole reason of existence of Bitcoin, Ethereum Classic and other true blockchains is human flourishing through trust minimization. 
+
+Indeed, proof of work actually helps the environment as an externality much more than proof of stake because renewable energy is cheaper than fossil fuel electricity, and miners are constantly seeking to lower their costs so they naturally move to renewable energy sources, providing cash flow to that segment of the industry and promoting its growth. 
+
+As Social Consensus believes in human agency, but human agency is prone to cheating, then penalties through slashing are required. Also, forks to reverse the chain, confiscate funds, slash bad actors, and change the history of the chain by majority voting are perfectly fine in that philosophy. If trust in intermediaries and human intervention were never the problem, then why was "trust minimization" even a  concern?
+
+Of course, Social Consensus has very little awareness of the perils that their actions and intervention will bring. It is unlikely that any actor in the network that is slashed by social consensus will passively accept a significant loss of their capital without a fight. Lawsuits and legal bureaucracy will be the name of the game in the Social Consensus world.
+
+In Social Consensus philosophy, "finality" is defined as a 66% vote of validators. This seems to be strong for the naive eye, but it is just a mirage. It really means that with only 33% of bad actors the network may be disrupted. This has been the vulnerability that has existed in computing networks since they were invented.
```

Comment:
Line 86: Correct preposition usage; Comma added
```suggestion
In Social Consensus philosophy, "finality" is defined as a 66% vote of validators. This seems to be strong to the naive eye, but it is just a mirage. It really means that with only 33% of bad actors, the network may be disrupted. This has been the vulnerability that has existed in computing networks since they were invented.
```